lib/kcl/workers/consumer.rb

```diff
@@ -1,9 +1,13 @@
+require 'aws-sdk-kinesis'
+
 module Kcl::Workers
   # Shard : Consumer = 1 : 1
   # - get records from stream
   # - send to record processor
   # - create record checkpoint
   class Consumer
+    LEASE_RETRY=3
```

Comment:
this looks like it might also be better as a tunable configuration setting (so clients can override this arbitrary limit without needing `const_set`)